tests/api2/test_rsync_ssh_authentication.py

```diff
@@ -229,6 +228,7 @@ def test_ssh_credentials_key(cleanup, localuser, remoteuser, src, dst, ssh_crede
         "ssh_credentials": ssh_credentials["credentials"]["id"],
         "mode": "SSH",
         "remotepath": dst,
+        "ssh_keyscan": True,
```

Comment:
Why? Should not be necessary because we are using a keychain credential that already contains the host key.